showcase/src/app/shared/effects/stark-error-handling.effects.ts

```
@@ -30,6 +38,12 @@ export class StarkErrorHandlingEffects {
 		return this.actions$.pipe(
 			ofType<StarkUnhandledError>(StarkErrorHandlingActionTypes.UNHANDLED_ERROR),
 			map((action: StarkUnhandledError) => {
+				console.log("ERROR: ", action.error.toString());
+
+				if (IGNORED_ERRORS.includes(action.error.toString().trim())) {
```

Could we move this logic in stark-core ?

---

I removed this code, I did a bit more digging and found another (more correct) solution.
But is this functionality we want in the core? (Ignoring errors) Because I think I could make something cleaner in that case. 😄

---

Indeed, we prefer not to ignore errors as you suggest.
What is your "more correct" solution ? 😊

---

A polyfill 🙈

---

I think the polyfill is the right solution 😊